config/settings_data.json

```diff
@@ -324,14 +292,12 @@
       "color_add_wishlist_modal_subtitle_bg": "#F3F3F3",
       "color_add_wishlist_modal_delete_icon": "#EC6A5E",
       "color_branch_closed": "#EC6A5E",
-      "color_add_to_cart_in_cart_bg": "#F3F3F3",
-      "color_add_to_cart_in_cart": "#757575",
-      "color_product_icon": "#D9D5CE",
-      "color_product_icon_active": "#F0AD4E",
       "color_product_outline": "#EDEDED",
+      "color_price": "#3C763D",
```

Comment:
Why do we introduce new and unused variable?

Reply:
This line was moved here from the beginning of the list.